src/index.html

```diff
@@ -7,12 +8,61 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
+  <!-- #endregion head-->
   <body>
-    <h1>Product cards</h1>
+    <article
```

Comment:
Full height of card with including of indents(paddings) should be ~400px, but without hardcode of this value. At the moment you have 198.4 + 33 + 17 + (0.8 * 2).
![image](https://github.com/mate-academy/layout_product-cards/assets/64101898/ad0348c1-d247-4c8b-a88b-9e1ccce3416c)